Community-Committee-Charter.md

```diff
@@ -90,36 +83,11 @@ If an agenda item cannot reach a consensus a CommComm member can call for either
 
 ## Section 9. Roles
 
-Node.js Foundation git repositories for Community Committee Initiatives
-are maintained by the CommComm and additional Collaborators who are added by
-the CommComm on an ongoing basis.
-
-Individuals making significant and valuable contributions,
-“Contributor(s)”, are made Collaborators and given commit-access to the
-project. These individuals are identified by the CommComm and their addition
-as Collaborators is discussed during the weekly CommComm meeting.
-Modifications of the contents of the git repository are made on a
-collaborative basis as defined in the development process.
-
-Collaborators may opt to elevate significant or controversial
-modifications, or modifications that have not found consensus to the CommComm
-for discussion by assigning the `cc-agenda` tag to a pull request or
-issue. The CommComm should serve as the final arbiter where required. The CommComm
-will maintain and publish a list of current Collaborators by Project, as
-well as a development process guide for Collaborators and Contributors
-looking to participate in the development effort.
-
-Individual Membership Directors of the Node.js Foundation are the Node.js
-project’s community voice on the board. There are two individual directors
-that sit on the Node.js Foundation board and they each serve a two-year term.
-Each Individual Membership Director is responsible for soliciting feedback and
-data that represents the wishes of other individual members and the community
-at large. These directors participate and observe CommComm meetings in order to deliver
-monthly updates to the Board of the project. They deliver monthly updates to the
-CommComm on what the Board is doing. They have been entrusted with the duty to make
-decisions based on the information they receive to best represent the community,
-and can gather input for proposals when relevant and granted permission to do
-so by the Board.
+OpenJS Foundation git repositories for Community Committee Initiatives are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Comment:
Node sure this should be OpenJS Foundation as the repos are in the Node.js org.  This is what is says in the updated TSC charter: 

>The Node.js git repository is maintained by the TSC and additional Collaborators who are added by the TSC on an ongoing basis.

https://github.com/nodejs/TSC/blob/master/TSC-Charter.md#section-8-project-roles

Reply:
How does this sound:

```suggestion
The Node.js Community Committee git repository and various initiative repositories are are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Reply:
Small nitpicks:

```suggestion
The Node.js git repositories for Community Committee and its initiatives are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Reply:
Tiny additional nit on your enhancement @ahmadawais:

```suggestion
The Node.js git repositories for the Community Committee and its initiatives are maintained by the CommComm and additional Collaborators who are added by the CommComm on an ongoing basis.
```

Reply:
Looks good